docs/data/data-grid/custom-columns/EditingWithDatePickers.tsx

```diff
@@ -70,7 +70,7 @@ function GridEditDateCell({
   field,
   value,
   colDef,
-}: GridRenderEditCellParams<any, Date | string | null>) {
+}: GridRenderEditCellParams<any, Date | null, string>) {
```

Comment:
Is it a fix of something?

Reply:
Yes, the date in the pickers can no longer be a string since v6 and the formatted value on the other hand is always a string

Reply:
Yes, this looks good to me 👍🏻